docs/transpiler.md

```diff
@@ -111,16 +111,16 @@ then `load()` from your own macro rather than from `@aspect_rules_ts`.
 
 ## Macro expansion
 
-When `transpiler` is used, then the `ts_project` macro expands to these targets:
+When `transpiler` or `declaration_transpiler` is used, then the `ts_project` macro expands to these targets:
```

Comment:
nit: `When 'transpiler' or 'declaration_transpiler' is set` is more accurate since from reading the code the macro expansion is when they are set and not necessarily if they are used
nit: looks like if `no_emit` is set the macro will also expand to these targets (minus the `[name]_types` target